src/components/SwiperCard.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+    <div>
+        <img :src="productImages[this.currentSrc]"  class="h-80 w-96"/>  
+        <button @click="prev()" class="absolute top-0 my-36"><i class="fa fa-arrow-circle-left text-3xl bg-white w-10 rounded-full" aria-hidden="true"></i></button>
+        <button @click="next()" class="absolute top-0 right-0 my-36"><i class="fa fa-arrow-circle-right text-3xl bg-white w-10 rounded-full" aria-hidden="true"></i></button>
+
+    </div>
+
+</template>
+<script >
+
+export default({
+    data(){
+        return{
+            currentSrc: 0,
+        }
+    },
+    props:{
+       productImages: {},
```

Comment:
we can also validate props. https://vuejs.org/guide/components/props.html#prop-passing-details

Reply:
done